cleanup.sh

```diff
@@ -1,25 +1,42 @@
+#!/bin/bash
 #remove directory
 
 cd ~
 
-rm -rf antora 
+rm -rf antora/ 
```

Comment:
why the addition of "/" ?

Reply:
This was likely an autopilot addition I added without really thinking about it. Both work since antora is a folder.

Reply:
@kelvinspencer is this resolved?

Reply:
it can be, the forward slash isnt necessary though, delete it if its not too much of an issue